assets/js/map.js

```diff
@@ -0,0 +1,203 @@
+(function(exports) {
+
+  var map = d3.select("#team-map")
+        .classed("loading", true),
+      status = map.append("p")
+        .html("Loading the map...");
+
+  // names and geographic locations of airport codes
+  var locations = [
```

Comment:
How will a new developer add these down the line? Does there need to be instructions for them?